src/status_im/ui/screens/network_settings/edit_network/events.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.ui.screens.network-settings.edit-network.events
+  (:require [re-frame.core :as re-frame]
+            [status-im.utils.handlers :refer [register-handler] :as handlers]
+            [status-im.utils.handlers-macro :as handlers-macro]
+            [status-im.ui.screens.accounts.utils :as accounts.utils]
+            [status-im.utils.ethereum.core :as ethereum]
+            [status-im.utils.types :as types]
+            [clojure.string :as string]))
+
+(defn- new-network [{:keys [random-id] :as cofx} network-name upstream-url type]
+  (let [data-dir (str "/ethereum/" (name type) "_rpc")
+        config   {:NetworkId      (ethereum/chain-keyword->chain-id type)
```

Comment:
is there a reason to choose camel-case naming for keywords despite to clojure style guide conventions?

Reply:
Nevermind, it was already the case before.

Reply:
Yup :) seems to be because of json serialization to status-go or something